test/e2e/servicecatalog/podpreset.go

```diff
@@ -22,7 +22,7 @@ import (
 	"time"
 
 	"k8s.io/api/core/v1"
-	settings "k8s.io/api/settings/v1alpha1"
+	settings "pkg/apis/settings"
```

Comment:
Why is the import path changed?

Reply:
In this PR, we'll need to make this test a feature test, by adding `[Feature:PodPreset]` in the test name, then the test won't run in pull-kubernetes-e2e-gce-etcd3.

In another PR, we need to add the test to the alpha suite, https://github.com/kubernetes/test-infra/blob/52b103fadb98e290d67fefeba6051eea8110fc46/jobs/config.json#L639.

cc @droot

Reply:
@caesarxuchao I think I have fixed the issue now, when you have time, could you look it over again?

Thanks